Interest groups limit section added to Privacy Sandbox FAQ (#5616)
GoogleChrome/developer.chrome.com: site/en/docs/privacy-sandbox/faq/index.md

*  `auctionSignals`, a property of the auction configuration object passed to `navigator.runAdAuction()`
*  `userBiddingSignals`, a property of the interest group configuration
   object passed to `navigator.joinAdInterestGroup()`
   

> ### What is the limit to the number of interest groups that a group owner (origin) can add a single browser to?

Reviewer: ```suggestion
### What is an owner's interest group limit for a single browser? {: #interest-group-limit }
```

Reviewer 2: Maybe something like this: 'What is the maximum number of interest groups per group owner for a single user?'

(The use of 'browser' is good, to emphasise that PS APIs work with the 'browser' not the 'user', but in this context it might seem to imply a 'browser' in the sense of Chrome versus Edge.)

> Chrome allows up to 1000 interest groups per owner, and up to 1000 interest group owners.

Reviewer: ```suggestion
Chrome allows up to 1000 interest groups per owner (origin) and up to 1000 interest group owners.
```
